lib/waitingList/index.js

```diff
@@ -1,61 +1,27 @@
 const mongoose = require('mongoose');
 const moment = require('moment');
 const Schema = mongoose.Schema;
-const waitingListSchema = new Schema({
-  departureDate: String,
-  pickupLoc: {
-    required: true,
-    type: [Number],
-    index: '2dsphere',
-  },
-  dropOffLoc: {
+const member = new Schema({
+  driverId: {
+    type: Schema.Types.ObjectId,
     required: true,
-    type: [Number],
-    index: '2dsphere',
   },
+  departureTime: Date,
+  createdOn: { type: Date, default: Date.now },
+});
+const waitingListSchema = new Schema({
+  departureDate: String,
```

Comment:
my bad, departureDate supposed to be removed on this PR, good catch! I will update the PR today, thanks